configs/repvgg/metafile.yml

```diff
@@ -0,0 +1,219 @@
+Collections:
+  - Name: RepVGG
+    Metadata:
+      Training Data: ImageNet
+      Epochs: 200
```

Comment:
I notice that some models use 120 epochs. If `Epochs` is not a shared field of all models, please move it to each model.